core/modules/css/KvInterfaceImplMem.cc

```diff
@@ -194,12 +206,35 @@ KvInterfaceImplMem::getChildren(string const& key) {
 void
 KvInterfaceImplMem::deleteKey(string const& key) {
     LOGF_DEBUG("deleteKey(%1%)." % key);
+
+    if (_readOnly) {
+        throw ReadonlyCss();
+    }
+
     if ( ! exists(key) ) {
         throw NoSuchKey(key);
     }
     _kvMap.erase(key);
 }
 
+std::string KvInterfaceImplMem::dumpKV() {
+    std::string result;
+    for (auto& pair: _kvMap) {
+        if (not result.empty()) result += '\n';
```

Comment:
I thought I was writing Python :) 
We have discussed that already some time ago, I think nobody had violently objected using "Python-like" operators. And I'm not alone doing it actually. Does it make it harder to read?